SwiftUI Piano Recording Task/SwiftUI Piano Recording Task/Instrument.swift

```diff
@@ -0,0 +1,55 @@
+import AVFoundation
+
+public typealias MIDINumber = Int
+public typealias NoteRange = CountableRange<MIDINumber>
+
+extension MIDINumber {
+    var pitchClass: MusicalNote {
+        let rawValue = self % MusicalNote.allCases.count
+        let musicalNote = MusicalNote(rawValue: rawValue)
```

Comment:
nit: would put the `!` at the end of this statement rather than on the next line.
This was copied from some of the earliest Swift code we wrote and we've since changed conventions